routing/record/validation.go

```diff
@@ -100,3 +112,18 @@ func CheckRecordSig(r *pb.Record, pk ci.PubKey) error {
 	}
 	return nil
 }
+
+type RecordSorter []*pb.Record
+
+func (rs RecordSorter) Less(i, j int) bool {
+	mv, _ := MoreValid(rs[j], rs[i])
```

Comment:
On Sat, Apr 18, 2015 at 10:45:57PM -0700, Jeromy Johnson wrote:
> note: records arent necessarily namespaced to their creator, there
> is shared dht space. For example, i can push anything I want to the
> key "cat".

Are keys like this used for anything important?  I don't see how you'd
avoid a bunch of writers clobbering each-other if they're all pushing
to the same (not-namespaced) key.